packages/lib/server/i18n.ts

```diff
@@ -4,17 +4,26 @@ import { WEBAPP_URL } from "@calcom/lib/constants";
 
 const i18nInstanceCache = new Map<string, any>();
 
+async function loadFallbackTranslations() {
+  try {
+    const res = await fetch(`${WEBAPP_URL}/static/locales/en/common.json`);
+    if (res) {
+      return await res.json();
+    }
+  } catch {
+    console.error("Could not fetch fallback translations.");
+  }
+  return;
```

Comment:
NIT: I would return `return {}` in `loadFallbackTranslations` instead of having `return fallbackTranslations ?? {};` below.